modules/ROOT/pages/administer-community.adoc

```diff
@@ -4,8 +4,11 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Administering your Anypoint API Community Manager community requires monitoring important notifications about new releases and downtime, upgrading API Community Manager, and if necessary, getting support from MuleSoft.
+Administering your API Community Manager community requires performing basic maintenance of your ACM Salesforce organization, monitoring important notifications about new ACM package releases and downtime, upgrading ACM, and if necessary, getting support from MuleSoft.
```

Comment:
1. About "performing basic maintenance of ...": it should be "performing basic maintenance on ...".
2. I know this bit wasn't one of your changes, but "monitoring important notifications" means that a set of notifications is already known to you and you are watching the performance of those notifications or other characteristics of them. Perhaps what you meant was "checking for important notifications"? The link text below also uses "monitor" when really what is meant is "Check for".
3. The "if necessary" part at the end of the sentence contradicts the verb of the sentence, which is "requires". So, the optional part should go into a separate sentence.
4. The sentence uses "upgrading", but the link text below uses "update". Those are two different types of task.

Reply:
These are good points!

1. Fixed.
2. Yes, I think "check for" is better. Fixed.
3. Fixed.
4. Fixed.